kubernetes-extension-test/src/test/resources/istio/gateway/.gitignore

```diff
@@ -0,0 +1 @@
+kubernetes
```

Comment:
Shall we add a new line at the end of the file?

Reply:
Fixed with https://github.com/ballerinax/kubernetes/pull/220/commits/5f8887c1388816ebe18d4066c627a7d43a88bea9